app/components/Sidebar.scss

```diff
@@ -0,0 +1,8 @@
+@import '../styles/app.global.scss';
```

Comment:
I changed it and now I'm importing only files with variables / mixins if there's such a need. I tried to load them as modules using recommended `@use` instead of `@import` but it doesn't work. It may be something linked to sass-loader or webpack, so it would take too much time to investigate it now.